source/guides/network/wifi.rst

```diff
@@ -102,8 +104,119 @@ TUI (Text-based User Interface)
 
    .. figure:: /_figures/wifi/nmtui_5.png
 
+wpa_supplicant
+**************
+
+Make sure NetworkManager is either disabled or not installed before using
```

Comment:
```suggestion
Make sure NetworkManager is either stopped, disabled by masking the service, or not installed
```
Since the service is system distributed and enabled by default, it needs to be masked instead of disabled. 
IIRC uninstalling is difficult.